client/web/src/components/login-toggle.tsx

```diff
@@ -162,7 +162,19 @@ function LoginPopoverContent({
       </div>
       {!auth.canManageNode && (
         <>
-          {!auth.viewerIdentity ? (
+          {auth.serverMode === "readonly" ? (
+            <p className="text-gray-500 text-xs">
+              This web interface is running in read-only mode.{" "}
+              <a
+                href="https://tailscale.com/s/web-client-connection"
```

Comment:
Anything set up over there yet?

Reply:
Not specific to read-only mode.  I'll actually go ahead and make this a dedicated short link for readonly mode, and then work on a PR to add docs before the 1.60 release.